app/views/users/_form.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_for @user, url: signup_path do |f| %>
```

Comment:
Dùng form_tag và form_with trong trường hợp này, in parameter lên đây nhé
So sánh các loại này nhé

Reply:
Dùng form tag khi không có đối tượng, thường dùng cho form search, lọc
Dùng form for cần có đối tượng để đặt vào vòng for, dùng cho tạo mới, sửa
Dùng form with khi có đối tượng hoặc không có đối tượng đều được

Reply:
Đối với form_tag:
 name"=>"Tạ Duy Hiếu", "email"=>"dalkdm@gmail.com", "password"=>"", "password_confirmation"=>""

Đối với form_with:
"user"=>{"name"=>"ta.duy.hieu@sun-asterisk.com", "email"=>"", "password"=>"123", "password_confirmation"=>""}